_data/swedlowlab.json

```diff
@@ -0,0 +1,216 @@
+{
+  "@context": "https://schema.org",
+  "@type": "Organization",
+  "member": [
+    {
+      "@type": "Person",
+      "givenName": "Jason",
+      "familyName": "Swedlow",
+      "jobTitle": "Co-Founder",
+      "description": "Jason Swedlow earned a BA in Chemistry from Brandeis University in 1982. He performed his PhD in Biophysics with Profs D. A. Agard and J. W. Sedat, finishing in 1994. Dr Swedlow was a postdoctoral fellow at UCSF and then Harvard Medical School from 1994 and 1998, supported by a Damon Runyon Walter Winchell Cancer Research Fund Fellowship from 1995 to 1997. In 1998, Dr Swedlow established his own laboratory at the Wellcome Trust Biocentre, University of Dundee, Scotland as a Principal Investigator and Wellcome Trust Career Development Fellow. He was awarded a Wellcome Trust Senior Fellowship in 2002, and named Professor of Quantitative Cell Biology in 2007. His lab focuses on studies of mitotic and interphase chromosome structure and dynamics. He is Co-Director of the Analytical and Quantitative Microscopy Course. He is co-founder of OME (along with Peter Sorger and Ilya Goldberg)."
```

Comment:
Line length is probably the major downside of the JSON strategy,no ?

Reply:
Definitely, I tried investigating a bit into https://gdevops.gitlab.io/tuto_git/tools/pre-commit/repos_hooks/repo_pre_commit_hooks/pretty-format-json/pretty-format-json.html but I could not find a built-in option to set a max-line-length and break a long string into an array representation.